specs/IR-0006/suite/test_spec_ir0006.py

```
@@ -0,0 +1,174 @@
+"""IR-0006 spec 结构自测（suite/——T-14：spec PR 必含非空测试文件且含真实断言）。
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)